include/env.h

```diff
@@ -0,0 +1,38 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   env.h                                              :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: yliu <yliu@student.42.jp>                  +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/09/08 12:35:50 by yliu              #+#    #+#             */
+/*   Updated: 2024/09/10 14:58:06 by yliu             ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef ENV_H
+# define ENV_H
+
+# include <stdlib.h>
+# include <unistd.h>
+
+# include "libft.h"
+# include "utils.h"
+
+typedef struct s_env
+{
+	char		*name;
```

Comment:
Consider using 'key' instead of 'name' for environment variables like hash map.

Reply:
beb87c1 Change hash map structure field variable: name -> key